packs/equipment/silver-orb-powder.json

```diff
@@ -0,0 +1,100 @@
+{
+    "_id": "kG26J0AfQAVe9FYs",
+    "img": "systems/pf2e/icons/equipment/alchemical-items/alchemical-bombs/thunderstone.webp",
+    "name": "Silver Orb (Powder)",
```

Comment:
Should this be Moderate rather than Powder?

Reply:
This one is in the PDF as "silver power orb" instead of silver orb. The other errors are form Paizo getting us a PDF with encoding errors. Every `i` or `l` following an i, l or f was stripped and I didn't catch them all. This is annoyingly common for pre-release stuff.